packages/react/src/primitives/CheckboxField/CheckboxField.tsx

```diff
@@ -43,28 +28,11 @@ export const CheckboxField: Primitive<CheckboxFieldProps, 'input'> = ({
       testId={testId}
     >
       <Checkbox
-        alignContent={alignContent}
```

Comment:
I'm assuming we don't need any of these because of the rest spread? Much cleaner!

Reply:
Yup, correct. Just let the rest spread take care of those